sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClient.java

```diff
@@ -518,9 +522,12 @@ public Secret restoreSecret(byte[] backup) {
      * <p>Lists the deleted secrets in the key vault and for each deleted secret prints out its recovery id.</p>
      * {@codesnippet com.azure.security.keyvault.secretclient.listDeletedSecrets}
```

Comment:
This should have the samples header as well and at other places too.
`* <p><strong>Samples to iterate DeletedSecret</strong></p>`